taar/recommenders/legacy_recommender.py

```diff
@@ -0,0 +1,51 @@
+from ..recommenders import utils
+
+ADDON_LIST_BUCKET = 'telemetry-private-analysis-2'
+ADDON_LIST_KEY = 'taar/legacy/legacy_dict.json'
+
+
+class LegacyRecommender:
```

Comment:
Could you please add a logger here as done with the other recommenders? Let's also add log lines where you think it's useful